docs/development/contributing_guide.md

```diff
@@ -76,22 +83,43 @@ cd path/where/you/have/cloned/the/repository
 poetry install --extras all
 ```
 
-This will install the runtime and developer dependencies for the project.
+This will install the runtime and developer dependencies for the project.  Be sure to check
+that the `poetry install` step completed cleanly.
 
 ## Running Synapse via poetry
 
 To start a local instance of Synapse in the locked poetry environment, create a config file:
 
 ```sh
 cp docs/sample_config.yaml homeserver.yaml
+cp docs/sample_log_config.yaml log_config.yaml
 ```
 
-Now edit homeserver.yaml, and run Synapse with:
+Now edit `homeserver.yaml`, things you might want to change include:
+- setting a servername
+- fixing pathnames to be correct for your system
+- using a PostgreSQL database instead of SQLite
+- adding a [registration_shared_secret](https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html#registration_shared_secret)
+  for use with [register_new_matrix_user](https://matrix-org.github.io/synapse/latest/setup/installation.html#registering-a-user)
+
+edit `log_config.yaml` if necessary:
+- you may wish to set the root handler for logging in your dev environment to `console`
```

Comment:
I think we can remove this as it's not really necessary to change anything if you're just getting your dev setup going